site/docs/4.5.1/overview/releaseNotes.md

```diff
@@ -11,491 +11,16 @@ below.
 
 ## Highlights
 
-TODO
+- Ability to use only lookback interface on LocalBookie, see [https://github.com/apache/bookkeeper/issues/407](https://github.com/apache/bookkeeper/issues/407)
```

Comment:
don't think it worth a highlight. we can remove it.

Reply:
done